decentraland/unity-explorer
# Chore: analytics login flow extension

## What does this PR change?

Initial loading analytics was extended in the part of authentication flow by adding more events related to Authentication and Loading screens. New reactive property was presented in `AuthenticationController` to better reflect different login flows.

## How to test the changes?

1. Launch the explorer
2. Verify it loading to the GPlaza as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
